util/error.go

```diff
@@ -90,3 +90,16 @@ func (c ConflictError) Headers() map[string]string {
 	return map[string]string{}
 }
 
+type ServerError string
+
+func (a ServerError) Error() string {
+	return string(a)
+}
```

Comment:
What's the purpose of this? Why is a converted to a string? I assume that is what string(a) does, but ServerError is already a string.

Reply:
Okay, I realized that I might be reading the function signature in the wrong way. Go is confusing at times.